Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+pipeline {
```

Comment:
MAJOR: before merging, change jenkins pipeline to build master branch instead of this one